### PR TITLE
refactor(init): improve node version check

### PR DESCRIPTION
**Pull request by EvolutionX-10**

### CODE REVIEW

**Comment by jacoobes** [comment]
Is the console log needed? ( check review )

**Comment by EvolutionX-10** [comment]
Oh damn I forgot to remove that, let me remove

**Comment by EvolutionX-10** [comment]
@jacoobes removed, also your review wasn't submitted I think, or github doesnt show it to me

**Comment by EvolutionX-10** [comment]
@jacoobes reviewwwww as well as check https://github.com/sern-handler/awesome-plugins/pull/6 as well

File: src/commands/init.js [hunk 1]
```diff
@@ -24,11 +24,16 @@ const { prompt } = prompts;
  */
 export async function init(flags) {
 	// * Check if node version is valid
-	const node = await execa('node', ['--version']);
-	if (/v1(([0-6]\.[2-9])|([0-5]\.[0-9]))/gm.test(node.stdout)) {
+	const { version } = process;
+	console.log(version.split('.'));
+	const [ major, minor ] = version.split('.');
```
[the remark this answers is not in the document]
Comment (EvolutionX-10) [reply src/commands/init.js]: okay now i get it why its outdated, your review was pending same as mine in plugins repo 😂 
resolving this

[the remark this answers is not in the document]
Comment (jacoobes) [reply src/commands/init.js]: thats weird yeah i wonder why i commented twice

**Pull request merged**